### PR TITLE
IA-4

### PR DESCRIPTION
**Pull request by kevindoingbizas**

### CODE REVIEW

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 1]
```diff
@@ -217,7 +217,7 @@
 - control_key: IA-4
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
  implementation_status: not applicable
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 2]
```diff
@@ -233,14 +233,30 @@
...
+        understanding-identity-provider)
+        responsibility for authentication, session management, and OAuth
+        controls to the hosting environment’s LDAP and leverages CAC/PKI,
+        FIPS 140-2 compliant ciphers, SSO and does not provide for group
+        accounts.
+        
+        This system should be used to preventing reuse of identifiers. DOD
+        systems currently support this capability with the CAC card
+        edipi. 
+
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Preventing reuse of identifiers for an organization-defined
        time period is deferred to the
        organizational identity provider (such as
        LDAP, Active Directory, Red Hat SSO, etc).

        To aid system administrators, documentation on
        supported authentication providers can be found in the
        OpenShift product documentation:

        https://access.redhat.com/documentation/en-us/
        openshift_container_platform/latest/html/authentication/
        understanding-identity-provider
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 3]
```diff
@@ -233,14 +233,30 @@
...
+        OCP offloads (via: https://access.redhat.com/documentation/en-us/
+        openshift_container_platform/4.3/html/authentication/
+        understanding-identity-provider)
+        responsibility for authentication, session management, and OAuth
+        controls to the hosting environment’s LDAP and leverages CAC/PKI,
+        FIPS 140-2 compliant ciphers, SSO and does not provide for group
+        accounts.
+        
+        The organization needs to define time period for disabling the 
+        identifier.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Disabling the identifier after an organization-defined
        time period of inactivity is deferred to the organizational
        identity provider (such as LDAP, Active Directory, Red Hat
        SSO, etc).

        To aid system administrators, documentation on
        supported authentication providers can be found in the
        OpenShift product documentation:

        https://access.redhat.com/documentation/en-us/openshift_container_platform/latest/html/authentication/understanding-identity-provider
```

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**